requirements.txt

```diff
@@ -5,7 +5,7 @@ chardet==3.0.4
 configparser==3.5.0
 dateparser==0.6.0
 docopt==0.6.2
-elasticsearch==6.2.0
+elasticsearch==6.4.0
```

Comment:
We're using ELK 6.7 atm, does this exist as python lib?

Reply:
6.4.0 appears to be the latest version https://pypi.org/project/elasticsearch/#history